### PR TITLE
Only languages filter scanner

### PR DESCRIPTION
**Pull request by smacker**
~Depends on https://github.com/src-d/lookout-sdk/pull/44~
Depends on a new `lookout-sdk` release containing [lookout-sdk#44](https://github.com/src-d/lookout-sdk/pull/44)

Fix: https://github.com/src-d/lookout-sdk/issues/36

Review commit by commit would be easier than all the changes together.

### CODE REVIEW

File: service/enry/scanner.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+func filterVendor(f *lookout.File) (bool, error) {
+	if f == nil {
+		return false, nil
+	}
+
+	return enry.IsVendor(f.Path), nil
+}
+
+// newChangeExcludeVendorScanner creates new FnChangeScanner
+func newChangeExcludeVendorScanner(scanner lookout.ChangeScanner) *lookout.FnChangeScanner {
```
[the remark this answers is not in the document]
Comment (dpordomingo) [reply service/enry/scanner.go]: it makes sense, thanks for the clarification !!

**Comment by dpordomingo** [comment]
The dependency on src-d/lookout-sdk#44 was already satisfied.
Is there something missing?

**Comment by dpordomingo** [comment]
Waiting for the new release of `lookout-sdk`

File: service/enry/scanner_test.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+			Head: &lookout.File{Path: "f1new.go"},
+		},
+		{
+			Base: &lookout.File{Path: "f1old.py"},
+		},
+		{
+			Base: &lookout.File{Path: "f2old.py"},
+			Head: &lookout.File{Path: "f2new.js"},
+		},
+		// the change that used to be vendor but isn't anymore should survive
```
Comment (carlosms) [inline service/enry/scanner_test.go]: Is this comment correct? The test case seems to be the exact opposite

Comment (smacker) [reply service/enry/scanner_test.go]: thanks! The comment is correct but the code is wrong, the right one was at the very end of use cases. Fixed.

**Pull request merged**